scripts/projector.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# failfast
+set -e
+
+# Define these 2 optional variables if you want to customize the default installation,
+# then copy-paste the remainder of the script:
+# - the name of the IDE to install (defaults to "IntelliJ IDEA Community Edition 2020.3.4")
+# Note: to get the list of available IDEs, launch `projector find`
+# projector_ide_name="IntelliJ IDEA Community Edition 2020.3.4"
+# - the password to protect access to projector (defaults to no password)
+# projector_password="class-specific-random-password"
+
+projector_ide_name=${projector_ide_name:-IntelliJ IDEA Community Edition 2020.3.4}  # Latest projector-tested IDE
+projector_config_name=strigo
+
+# Install projector-installer
+# https://github.com/JetBrains/projector-installer
+apt-get update
+apt-get install -y --no-install-recommends python3 python3-pip python3-cryptography less libxext6 libxrender1 libxtst6 libfreetype6 libxi6
+pip3 install projector-installer
+
+# Setup projector
+sudo -u ubuntu projector --accept-license ide autoinstall --config-name "${projector_config_name}" --ide-name "${projector_ide_name}" --hostname {{ .STRIGO_RESOURCE_DNS }}
+if [ -n "${projector_password}" ]; then
+  cat << EOF >> ~ubuntu/.projector/configs/${projector_config_name}/config.ini
+[PASSWORDS]
+password = ${projector_password}
+ro_password = ${projector_password}
+EOF
+  sudo -u ubuntu projector --accept-license config rebuild "${projector_config_name}"
+fi
+
+# Create, enable and start projector config service
```

Comment:
After further testing, I found issues with keyboard shortcut and copy/past.
This issues has been addressed with secure configuration.

```
sudo snap install --classic certbot
sudo ln -s /snap/bin/certbot /usr/bin/certbot
sudo certbot certonly --standalone -d {{ .STRIGO_RESOURCE_DNS }} --staple-ocsp -m strigo@zenika.com --agree-tos

sudo cp /etc/letsencrypt/live/{{ .STRIGO_RESOURCE_DNS }}/fullchain.pem /home/ubuntu
sudo cp /etc/letsencrypt/live/{{ .STRIGO_RESOURCE_DNS }}/privkey.pem /home/ubuntu
sudo chown ubuntu:ubuntu fullchain.pem
sudo chown ubuntu:ubuntu privkey.pem

sudo -u ubuntu projector install-certificate "${projector_config_name}" --certificate /home/ubuntu/fullchain.pem --key /home/ubuntu/privkey.pem
```

Reply:
I'll add the TLS option with #18 
https://zenika.slack.com/archives/C0110JT6TC1/p1630321340002300?thread_ts=1629899022.013500&cid=C0110JT6TC1

Reply:
Yes you're absolutely right, on the other hand, without TLS it's really hard to use projector-client

Reply:
I understand, yes.
Projector could propose an explicit insecure use of those features (but I'm not even sure it's possible).

Reply:
23c5046 to add option for TLS (created through #18)